common/display_model/src/test/scala/weco/catalogue/display_model/image/DisplayImageTest.scala

```diff
@@ -0,0 +1,27 @@
+package weco.catalogue.display_model.image
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import weco.catalogue.display_model.locations.DisplayDigitalLocation
+import weco.catalogue.internal_model.generators.ImageGenerators
+
+class DisplayImageTest extends AnyFunSpec with Matchers with ImageGenerators {
+  it(
+    "sets the thumbnail to the first iiif-image location it finds in locations") {
+    val imageLocation = createImageLocation
+    val image = createImageDataWith(
+      locations = List(createManifestLocation, imageLocation)).toIndexedImage
```

Comment:
Is it important that it's the first iiif-image location?

Should this location list have more than one location in it to ensure that it does find the first?

Reply:
Good suggestion, will change.